charts/k8s-monitoring/docs/Customizations.md

```diff
@@ -0,0 +1,107 @@
+  [discovery.relabel](https://grafana.com/docs/agent/latest/flow/reference/components/discovery.relabel/) component. They filter from all Nodes, services, endpoints, or Pods on the
+Cluster to a specific one that has the metrics. 
+  Use these rules to target a specific service.
+* `metrics.extraRelabelingRules` - Same as above, but these rules are applied to all metric sources.
+* `metrics.<source>.labelMatchers` - Defines what Kubernetes labels need to match on the target.
+* `metrics.<source>.service.port` - Defines the name of the port that has the metrics endpoint. 
+
+### Scraping customizations
+
+These fields affect how metric targets are scraped. Typically, you'll make changes here if 
+you want to change something about how often to scrape or which target to scrape.
+
+* `metrics.<source>.scrapeInterval` - How often to scrape metrics from a target.
+* `metrics.scrapeInterval` - Same as above, but changes the global default.
+* `metrics.<source>.service.isTLS` - Determines whether the Agent use HTTPS and TLS to scrape the target instead of HTTP.
+
+### Processing customizations
+
+By default, we already use an "allow list" to filter to a specific set of metrics, meaning the list drops metrics that are not useful for monitoring Kubernetes Clusters. The following fields allow for more customizations of the metrics after they have been scraped, but before being sent to the external metric service for storage. Typical changes that you might want to do here include setting, modifying, or dropping metric labels.
+
+* `metrics.<source>.extraMetricsRelabelingRules` - Rules that modify metrics and will populate the rules
+  section of a [prometheus.relabel](https://grafana.com/docs/agent/latest/flow/reference/components/prometheus.relabel/)
+  component. Use these rules to perform arbitrary modifications to metrics or metric labels.
+* `metrics.extraMetricsRelabelingRules` - Same as above, but the rules are applied to metrics from all metric sources.
+* `metrics.<source>.allowList` - Sets a list of metrics that will be kept, dropping any metrics that don't match.
+* `extraServices.prometheus.externalLabels` - A key-value set that defines labels and values to be set for all metrics
+  being sent. It sets the `external_labels` section of
+  [prometheus.remote_write](https://grafana.com/docs/agent/latest/flow/reference/components/prometheus.remote_write/#arguments)
+  component.
+
+### Delivery customizations
+
+These fields adjust how metrics are delivered to the metrics service. Typically, change these only to
+tune the performance of metrics uploads.
+
+* `extraServices.prometheus.processors` - These processors control the batch and memory size of metrics to bundle
+  before sending them to the service. Only applies to metrics services using the OTLP or OTLPHTTP protocols.
+* `extraServices.prometheus.wal` - Controls the behavior of the Write Ahead Log that's used in the remote_write
+  component.
+
+## Logs
+
+### Discovery customizations
```

Comment:
```suggestion
### Customize which logs to gather
```

Reply:
Probably need an intro sentence here - why would I want to customize log discovery?